tests/common.go

```diff
@@ -4595,6 +4606,70 @@ func CreateBucket(provider string, bucketName string) {
 	})
 }
 
+// IsBucketEmpty returns true if the bucket for a provider is empty
+func IsBucketEmpty(provider, bucketName string) (bool, error) {
```

Comment:
This function is confusing since we are using it for both Aws and Nfs. Can we define generic name ?

Reply:
Changed it to `IsBackupLocationEmpty`